dockerfiles/Dockerfile-toolchain

```diff
@@ -139,5 +141,17 @@ RUN curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash - \
 # easier.
 # RUN sudo ln -sf /usr/share/zoneinfo/UTC /etc/localtime
 
+# --- Build and install capnproto go compiler
+# This package was particularly tricky to get to install correctly for the version of go we use.
```

Comment:
I just wasn't comfortable making that decision in this PR, and I don't know how much it would help with this problem. I should update this comment, because I tried to install it using go 1.16 as well after I put this in place, but had to rely on this same trick because of the way this package is structured.